Frontend/android-native/app/src/main/java/in/juspay/mobility/MainActivity.java

```diff
@@ -349,7 +351,10 @@ protected void onCreate(Bundle savedInstanceState) {
             // TODO Auto-generated catch block
             e.printStackTrace();
         }
-
+        CleverTapAPI cleverTap = CleverTapAPI.getDefaultInstance(getApplicationContext());
+        CleverTapAPI.createNotificationChannel(getApplicationContext(),"NammaYatri","NammaYatri","notification",NotificationManager.IMPORTANCE_MAX,true);
```

Comment:
@ashkritidewan2000  Same code will be used for Yatri sathi and Yatri. Please cover those cases as well.